### DISCUSSION TITLE
俄烏戰爭練功場，美軍使用人工智慧分析戰事訓練系統能力

### DISCUSSION THREAD

**Discussion by Yao900108** [1101-GS4719-C/Python-Community-GS4719-C#567] [科技新知]
烏俄戰爭開打至今，雖然標槍、刺針飛彈等各種援助烏克蘭軍的武器大放風采，但美國國防部近期表示，藉由機器學習和人工智慧系統，美軍也從每場戰役分析大量資料和情報，學習俄羅斯的戰略和戰術運用。
目前美軍內部各種 AI 和機器學習系統，透過衛星、電子偵察機和各種情報來源，取得非常大量資料，對協助美軍分析俄軍戰術及提升系統精確度都有很大幫助。
等戰爭結束後，五角大廈會總體分析所有戰爭期間資料，屆時可更詳細評估俄軍真實戰力、美軍後續因應和軍力分佈等，提供國會和決策單位更多參考資料。

https://technews.tw/2022/04/26/usa-ai-systems-have-already-learnt-alot-from-russian-ukrainian-war/

**Comment by HuanYuuuuuu** [comment]
深偽技術用在資訊情報戰上感覺也會相當的可怕

**Comment by cordydeng** [comment]
這項技術如果能夠馬上實用在烏俄戰爭上，一定可以對戰爭有很大的影響，也希望戰爭早日結束
